experimental/casemapping/tests/gen_greek_to_me.rs

```diff
@@ -0,0 +1,146 @@
+                        data.ypogegrammeni = true;
+                    }
+                    // Ignore all small letters
+                    '\u{1D00}'..='\u{1DBF}' | '\u{AB65}' => (),
+                    // caps: [[:Grek:]&[:L:]-[\u1D00-\u1DBF\uAB65]] . NFD, remove non-letters, uppercase
+                    letter if GeneralCategoryGroup::Letter.contains(gc.get(letter)) => {
+                        if data.uppercase.is_some() {
+                            panic!("Found multiple letters within decomposition of {ch}");
+                        }
+                        let letter = letter.encode_utf8(&mut encode_scratch);
+                        let uppercased = cm.uppercase_to_string(&letter, &und);
+                        let mut iter = uppercased.chars();
+                        let uppercased = iter.next().unwrap();
+                        assert!(
+                            iter.next().is_none(),
+                            "{letter} Should uppercase to a single letter char, instead uppercased to {uppercased:?}"
+                        );
+                        data.uppercase = Some(uppercased);
+                    }
+                    _ => (),
+                }
+            }
+        }
+
+        vec.push(data)
+    }
+
+    let data = CodePointTrieBuilderData::ValuesByCodePoint(&vec);
+    let trie: CodePointTrie<GreekPrecomposedLetterData> = CodePointTrieBuilder {
+        data,
+        default_value: GreekPrecomposedLetterData::default(),
+        error_value: GreekPrecomposedLetterData::default(),
+        trie_type: TrieType::Small,
+    }
+    .build();
+
+    let output = format!(
+        r#"// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+// This file is generated by running `cargo test --test gen_greek_to_me --features compiled_data,datagen
+//
+// Do not edit manually
+
+use super::GreekPrecomposedLetterData;
+use icu_collections::codepointtrie::CodePointTrie;
+
+pub(crate) const GREEK_DATA_TRIE: CodePointTrie<'static, GreekPrecomposedLetterData> = {};
+"#,
+        trie.bake(&Default::default())
+    );
+
+    let mut rustfmt = Command::new("rustfmt")
+        .arg("--config")
+        .arg("newline_style=unix")
+        .stdin(Stdio::piped())
+        .stdout(Stdio::piped())
+        .spawn()
+        .expect("spawn failed");
+    let mut rustfmt_stdin = rustfmt.stdin.take().unwrap();
+    write!(rustfmt_stdin, "{output}").expect("write failed");
+
+    drop(rustfmt_stdin); // EOF
+
+    let output = rustfmt
+        .wait_with_output()
+        .expect("execution of rustfmt failed");
+    if !output.status.success() {
+        let stderr = String::from_utf8(output.stderr).expect("rustfmt output not utf-8");
+        panic!("rustfmt failed: {stderr}");
+    }
+    let output = String::from_utf8(output.stdout).expect("rustfmt output not utf-8");
```

Comment:
Thought: this is duplicated in `baked_exporter`. We could add this as a convenience fucntion in the `databake` crate